### PR TITLE
docs: v7 update guide mention minimum dependencies

### LINKED ISSUE #1705: Docs: V7 update guide needs to mention minimum dependencies
### In our gitter support channel at https://gitter.im/ngrx/platform

ERROR in node_modules/@ngrx/store/src/models.d.ts(31,43): error TS1005: ';' expected. node_modules/@ngrx/store/src/models.d.ts(31,67): error TS1005: ',' expected.

### PR DESCRIPTION
**Pull request by go2theG**
Closes #1705

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #1705

## Does this PR introduce a breaking change?

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: projects/ngrx.io/content/guide/migration/v7.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ To update your packages to the latest released version, run the command below.
 ng update @ngrx/store
 ```
 
+## Dependencies
+
+NgRx now has a minimum version requirement on Angular 7.
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/migration/v7.md]:
```suggestion
V7 has the minimum version requirements:

- Angular version 7
- TypeScript 3.1.x
- RxJS 6.x

```

Comment (timdeschryver) [reply projects/ngrx.io/content/guide/migration/v7.md]: Nit, should we add "version" to every dependency or to none of the dependencies?

Comment (brandonroberts) [reply projects/ngrx.io/content/guide/migration/v7.md]: I know this is a point of emphasis on the Angular side. I'd say add to all of them

File: projects/ngrx.io/content/guide/migration/v7.md [hunk 2]
```diff
@@ -10,6 +10,15 @@ To update your packages to the latest released version, run the command below.
 ng update @ngrx/store
 ```
 
+## Dependencies
+
+V7 has the minimum version requirements:
+
+- Angular version 7
+- TypeScript 3.1.x
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/migration/v7.md]:
```suggestion
- TypeScript version 3.1.x
```

File: projects/ngrx.io/content/guide/migration/v7.md [hunk 3]
```diff
@@ -10,6 +10,15 @@ To update your packages to the latest released version, run the command below.
...
+- RxJS 6.x
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/migration/v7.md]:
```suggestion
- RxJS version 6.x
```

**Comment by brandonroberts** [comment]
Thanks @go2theG!

**Pull request merged**